### PR TITLE
Fix test bed

### LINKED ISSUE #97: The neweat Testbed is broken
my Chrome  had the following results:
image

can anyone solove it?

### PR DESCRIPTION
**Pull request by huy-nguyen**
Adapt the solution from https://github.com/kripken/box2d.js/issues/87#issuecomment-304304620 and fix script loading error to make the test bed work again.

Fixes #97.

### CODE REVIEW

File: demo/html5canvas/testbed.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
         <script type="text/javascript">  
             var Box2D;
             if (!Box2D) Box2D = (typeof Box2D !== 'undefined' ? Box2D : null) || Module;
+            var Box2DConstructor = Box2D
```
Comment (kripken) [inline demo/html5canvas/testbed.html:24]: looks like this is never used?

File: demo/html5canvas/testbed.html [hunk 2]
```diff
@@ -21,6 +21,9 @@
...
+            Box2D = Box2D()
+            Module = Box2D
```
Comment (kripken) [inline demo/html5canvas/testbed.html:26]: this also doesn't look used?

File: demo/html5canvas/testbed.html [hunk 3]
```diff
@@ -21,6 +21,9 @@
         <script type="text/javascript">  
             var Box2D;
             if (!Box2D) Box2D = (typeof Box2D !== 'undefined' ? Box2D : null) || Module;
+            var Box2DConstructor = Box2D
+            Box2D = Box2D()
```
Comment (kripken) [inline demo/html5canvas/testbed.html:25]: please add `;` on each line

File: demo/html5canvas/testbed.html (hunk 2 above)
Comment (huy-nguyen) [reply demo/html5canvas/testbed.html]: I removed line 24 (`Box2DConstructor`) but if I remove the other 2 lines, I get this error message:
```
Uncaught TypeError: Box2D.JSDraw is not a constructor
    at getCanvasDebugDraw (embox2d-html5canvas-debugDraw.js:85)
    at init (embox2d-html5canvas-testbed.js:275)
    at window.onload (testbed.html:27)
```
I think those 2 lines are essential to the working of the test bed.

**Pull request merged**